### PR TITLE
Grocery Bud Assignment implementation complete

### PR DESCRIPTION
**Pull request by NStro03**
Grocery Bud Assignment implementation complete.
 Please Review @ishaan@flyfin.org

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,8 +2,107 @@ import React, { useState, useEffect } from 'react'
 import List from './List'
 import Alert from './Alert'
 
+let itemCreatedCount = 1;
```
Comment (Ishaan-flyfin) [inline src/App.js]: we can use https://www.npmjs.com/package/uuid for unique ids

File: src/App.js [hunk 2]
```diff
@@ -2,8 +2,107 @@ import React, { useState, useEffect } from 'react'
...
+  const [item2EditId, setItem2EditId] = useState(null);
+  // Alert State
+  const [alert, setAlert] = useState({ text: "", type: null, isShowing: false });
+  // const [showAlert, setShowAlert] = useState(false);
+
+  useEffect(() => {
+    localStorage.setItem('itemList', JSON.stringify(itemList));
+  }, [itemList]);
+
+  function getSavedList() {
```
Comment (Ishaan-flyfin) [inline src/App.js]: As this function is not related to this component, we can move it outside of it

**Review by Ishaan-flyfin — CHANGES_REQUESTED** [review]
minor comments

File: src/App.js (hunk 1 above)
Comment (NStro03) [reply src/App.js]: Done.

File: src/App.js (hunk 2 above)
Comment (NStro03) [reply src/App.js]: Moved it out of App and made it global.
